src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/OutgoingTransfers.cshtml

```diff
@@ -75,8 +96,13 @@
                 </table>
                 <br />
                 @{
-                    Html.RenderPartial("~/Views/Shared/_EvidenceNotesPagerPartial.cshtml", Model.EvidenceNotesDataList, new ViewDataDictionary() { { "pagerRouteValues", new { tab = Model.ActiveDisplayOption.ToDisplayString(), selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear } } });
+                    Html.RenderPartial("~/Views/Shared/_EvidenceNotesPagerPartial.cshtml", Model.EvidenceNotesDataList, new ViewDataDictionary()
+                        { { "pagerRouteValues", new { tab = Model.ActiveDisplayOption.ToDisplayString(),
+                        selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear,
```

Comment:
Can you check this works when changing the compliance year.

For the AA user there was a bug and had to update the selected compliance year to ,

manageEvidenceNoteViewModel_SelectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear

Reply:
true. I will update it.

Reply:
fixed.